sympy/ntheory/modular.py

```diff
@@ -253,3 +271,47 @@ def combine(c1, c2):
         if symmetric:
             return symmetric_residue(n, m), m
         return n, m
+
+
+def crt_cartesian(rem, mod):
```

Comment:
Even though the original input order is bit non-intuitive, I think swapping it for `crt_cartesian` is worse.